### DISCUSSION TITLE
Error resetting password: Could not modify attribute for DN [uid=lixiaokun, ou=People, dc=nodomain]  First of all, I ruled out the permission of the ldap management user

### DISCUSSION THREAD

**Discussion by lixiaokun777** [keycloak/keycloak#16135] [Q&A]
![6fd2b41c1f2517169d0d53225b2588b](https://user-images.githubusercontent.com/57617854/208840550-39d3b507-a0be-481a-ac0d-51be4544d933.png)
![2c75ad979d71202b355628599058a4f](https://user-images.githubusercontent.com/57617854/208840557-026c02a2-27fe-4403-9ccf-3cb57c643dcd.png)
![2ea0499a78cad6bcf03364d684f827f](https://user-images.githubusercontent.com/57617854/208840565-bce0c3a2-4cc9-477b-8666-93969878a619.png)
![8396e36abd2106ccee3975f4aa77742](https://u
![23da1ff1f2a1be042905a99627f7835](https://user-images.githubusercontent.com/57617854/208840583-c613fc1f-9f78-4e12-81dd-fd56418e1850.png)
ser-images.githubusercontent.com/57617854/208840
![821582b0133cd5f7011caa2ecb7ef9a](https://user-images.githubusercontent.com/57617854/208840587-a504e4bb-38f7-4ac5-b129-ab474809bd4c.png)
576-d85fc614-b38b-4add-a803-2cdb359f486a.png)

**Comment by Kuroky79** [comment]
Привет, как ты исправил в итоге?

**Comment by Dennis1993** [comment]
Same question, what is the result?
I have the same problem. The service account reset the password for the user but this error message appears. But the password is updated correctly after that in my Active Directory.

Any ideas?
